Automattic/simplenote-macos
# Window: Preventing zoom when double clicking the First Responder

### Fix
In this PR we're adjusting the Window's behavior, so that double clicking over the First Responder doesn't Zoom into the Window.

cc @eshurakov This one annoyed me for a while!!
Ref. #626 

### Test
1. Click over the Search Bar
2. Enter any random term
3. Double click over the Search Bar

- [ ] Verify the Window doesn't expand
- [ ] Verify the Search Keyword gets highlighted

### Release
`RELEASE-NOTES.txt` was updated in d49541530f44db14d259fde04cdac0d9304def62 with:

> Fixed a bug that Zoomed the window when double clicking on the Search Bar #841
